doc/running.md

```diff
@@ -34,24 +64,55 @@ helm repo update
 
 ## <a name="env"></a>Environment variables
 
-At a mininum, you need to set the following environment variable:
+The simplest solution is to ask for a copy of a working `.env` file from one of
```

Comment:
Emailing is not a great system. It's a bit of a social barrier and means you can't do this under time pressure, if the other dev is on holiday, or out of hours. If we could put it in a repo it would be better. It would allow us to track changes over time. Run tests with it. I can see there are no personal credentials in there, which is good. I can see there are a few system secrets and lots of implementation details that we might not want to share. How about the .env is checked into the [config repo](https://github.com/ministryofjustice/analytics-platform-config/) and encrypted with git-crypt like the other things?

Reply:
The DevOps Handbook puts it better:

> we want developers to run production-like environment on their own workstations, created on demand and self-serviced. By doing this, developers can run and test their code in production-like environments as part of their daily work, providing early and constant feedback on the quality their work. ... Instead of merely documenting the specifications of the production environment in a document or on a wiki page, we crate a common build mechanism that creates all of our environments, such as for development, test, and production. By doing this, anyone can get production-like environments in minutes, without opening up a ticket, let alone having to wait weeks. ... All our requirements are embedded, not in documents or as knowledge in someone's head, but codified in our automated environment build process.

So I think that sharing .env files is not good practice. And if that is the best method we have right now, then let's mark it as such in the readme, and encourage people to improve on it in future, and suggest improvements such as:

a) a 'dev' mode that lets you run, but doesn't require extra setup
b) instructions of how to get the values from AWS, Auth0 etc
c) automated setup (hard)

Reply:
See #806 for a place to discuss requirements for this further.